### PR TITLE
Add comment: syntex version must match pnet_macros

### PR DESCRIPTION
**Pull request by k4rtik**
From the discussion with @faern and levi on #libpnet

### CODE REVIEW

File: docs/using_packet.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ version = "0.1.0"
 authors = ["My Name <my.email@mydomain.com>"]
 build = "build.rs"
 [build-dependencies]
-syntex = "0.31.*"
+syntex = "0.42.*" # syntex version must match the version being used in pnet_macros/Cargo.toml
```
Comment (faern) [inline docs/using_packet.md]: Specifying a number here at all is just doomed to become outdated sooner or later. IMO it's better to say something like
`syntex = "X" # Where X must be the syntex version being used in pnet_macros/Cargo.toml`

Comment (k4rtik) [inline docs/using_packet.md]: That makes sense. Thanks for taking a look @faern.

Comment (mrmonday) [inline docs/using_packet.md]: Seems like the kind of thing that would be nice to have as part of a release script at some point in the future - we could automatically update the document with the right version.

**Comment by mrmonday** [comment]
@homu r+

Thanks for the fix!

**Pull request merged**